### ISSUE TITLE
Multiple amazon-ebs builds causes packer crash

### ISSUE THREAD

**Issue by Sleekwater** [mitchellh/packer#3753]
As requested, here's a crash.log
[crash.log.gz](https://github.com/mitchellh/packer/files/384250/crash.log.gz)

FYI, we run 3 instances of packer in parallel on the same machine (from different directories) to build different base images for Tomcat/Ubuntu, Tomcat/Redhat and Wildfly/Redhat

Restarting the build then worked fine without any changes, so it's not like it's a showstopper. 

Kev
p.s. Just wanted to say thanks for a fine bit of work in Packer - we use it loads!

**Comment by rickard-von-essen** [comment]
Host OS: Windows
Builder: `amazon-ebs`
```
2016/07/26 16:21:33 Packer.exe: 2016/07/26 16:21:33 Allowing 300s to complete (change with AWS_TIMEOUT_SECONDS)
2016/07/26 16:22:03 Packer.exe: 2016/07/26 16:22:03 [INFO] Not using winrm communicator, skipping get password...
2016/07/26 16:22:03 Packer.exe: 2016/07/26 16:22:03 [INFO] Waiting for SSH, up to timeout: 5m0s
2016/07/26 16:22:03 ui: ==> amazon-ebs: Waiting for SSH to become available...
2016/07/26 16:22:04 Packer.exe: panic: interface conversion: interface {} is **ec2.Instance, not *ec2.Instance
2016/07/26 16:22:04 Packer.exe: 
2016/07/26 16:22:04 Packer.exe: goroutine 30 [running]:
2016/07/26 16:22:04 Packer.exe: panic(0x12f20e0, 0xc0821181c0)
2016/07/26 16:22:04 Packer.exe: 	/usr/local/go/src/runtime/panic.go:481 +0x3f4
2016/07/26 16:22:04 Packer.exe: github.com/mitchellh/packer/builder/amazon/common.SSHHost.func1(0x3a90b88, 0xc082252630, 0x0, 0x0, 0x0, 0x0)
2016/07/26 16:22:04 Packer.exe: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/builder/amazon/common/ssh.go:19 +0x714
2016/07/26 16:22:04 Packer.exe: github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).waitForSSH(0xc082480e00, 0x3a90b88, 0xc082252630, 0xc082207680, 0x0, 0x0, 0x0, 0x0)
2016/07/26 16:22:04 Packer.exe: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:119 +0x6f0
2016/07/26 16:22:04 Packer.exe: github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).Run.func1(0x3a90628, 0xc08204e120, 0xc0822279d0, 0xc0822279e0, 0xc082480e00, 0x3a90b88, 0xc082252630, 0xc082207680, 0xc08224d340)
2016/07/26 16:22:04 Packer.exe: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:39 +0x7f
2016/07/26 16:22:04 Packer.exe: created by github.com/mitchellh/packer/helper/communicator.(*StepConnectSSH).Run
2016/07/26 16:22:04 Packer.exe: 	/Users/cbednarski/go/src/github.com/mitchellh/packer/helper/communicator/step_connect_ssh.go:41 +0x1c7
2016/07/26 16:22:04 ui error: Build 'amazon-ebs' errored: unexpected EOF
2016/07/26 16:22:04 Builds completed. Waiting on interrupt barrier...
2016/07/26 16:22:04 machine readable: error-count []string{"1"}
2016/07/26 16:22:04 ui error: 
==> Some builds didn't complete successfully and had errors:
2016/07/26 16:22:04 machine readable: amazon-ebs,error []string{"unexpected EOF"}
2016/07/26 16:22:04 ui error: --> amazon-ebs: unexpected EOF
2016/07/26 16:22:04 ui: 
==> Builds finished but no artifacts were created.
2016/07/26 16:22:04 waiting for all plugin processes to complete...
2016/07/26 16:22:04 c:\Packer\Packer.exe: plugin process exited
2016/07/26 16:22:04 c:\Packer\Packer.exe: plugin process exited
2016/07/26 16:22:04 c:\Packer\Packer.exe: plugin process exited
2016/07/26 16:22:04 c:\Packer\Packer.exe: plugin process exited
```

**Comment by miry** [comment]
I have multiple amazon-ebs builds. And sometime I see `Waiting` for SSH or sometime for Ami available. And after that it is stuck. Then only solution to kill process `packer plugin packer-builder-amazon-ebs`.  and i see next msg: `Build 'centos' errored: unexpected EOF`

**Comment by dpiddy** [comment]
Interface conversion issue dup of https://github.com/mitchellh/packer/issues/3496?

**Comment by rickard-von-essen** [comment]
This too. Closing in favour of #3496

**Comment by miry** [comment]
FYI: I fixed this problem in my case, before I used the homebrew to install packer. And after I built myself via golang it started working fine from `master`.
